src/Behat/Behat/Tester/Runtime/RuntimeStepTester.php

```diff
@@ -117,6 +118,11 @@ private function testDefinition(Environment $env, FeatureNode $feature, StepNode
             return new UndefinedStepResult();
         }
 
+        $definition = $search->getMatchedDefinition();
```

Comment:
If a definition has been found, we mark it as used even if it may be skipped, as we want to count skipped definitions as used

Reply:
Maybe worth having this as a code comment to remind us why it's in this sequence in future?